backend/schema/metadatacronjob.go

```diff
@@ -0,0 +1,37 @@
+package schema
+
+import (
+	"fmt"
+
+	"google.golang.org/protobuf/proto"
+
+	schemapb "github.com/TBD54566975/ftl/backend/protos/xyz/block/ftl/v1/schema"
+)
+
+type MetadataCronJob struct {
+	Pos Position `parser:"" protobuf:"1,optional"`
+
+	Cron string `parser:"'+' 'cron' Whitespace @((' ' | Number | '-' | '/' | '*' | ',')+)" protobuf:"2"`
```

Comment:
Unless there's a good reason, this should be converted into a fully parsed structure, not a string that has to be further parsed. The simplest way to do this is to have this be the canonical home of the parser, and reuse it in the go-runtime. This is the pattern we're following everywhere else.

Reply:
That's how I started out, but ended up changing it because:
- the cronjob service is reading/writing straight cron strings to the db table. It felt right to have cron package that was able to parse cron strings directly that could be used by the schema code for validation and by the service code for execution. And keeping the cron library separate would allow us to swap it out at somepoint if needed
- I also tried adding more annotations to the cron package such that MetadataCronJob struct could have:
    - `Pattern cron.Pattern 'parser:"'+' 'cron' @@" protobuf:"2"'`
    - I wasn't sure about this though... either we'd need protobuf annotations in the cron package, or CronJobMetada would have the structure but the protobuf would just have a string (which I'm not sure if that misses the point of having the structure)

Reply:
SGTM 👍

I think the first reason around strings isn't that compelling, as that can still be the case if it's fully parsed, but keeping the cron package completely separate rather than tightly coupling it to the schema _is_ a compelling argument.